lib/structuraid_core/elements/reinforcement/rebar.rb

```diff
@@ -20,6 +20,10 @@ def area
           rebar_area.to_f
         end
 
+        def mass
+          find_standard_mass(rebar_number: number)
+        end
```

Comment:
can you add a test for this method?

Reply:
right, MWNMRATU

https://www.notion.so/pradaing/Glosary-8cdffa6630da4cf2b393f157f840b756?pvs=4#cc09ffec6ff943e89d79ace04c7022b8